simulation-model/simulation-model_shacl.ttl

```diff
@@ -0,0 +1,317 @@
+            sh:maxCount 1 ;
+            sh:message "Validation of model.verification.report failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "model.verification.report"@en ;
+            sh:order 3 ;
+            sh:path simulation-model:model.verification.report ],
+        [ skos:example "true" ;
+            sh:datatype xsd:boolean ;
+            sh:description "Is the model validated, according to the validation concept"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of model.validation.status failed!"@en ;
+            sh:minCount 1 ;
+            sh:name "model.validation.status"@en ;
+            sh:order 4 ;
+            sh:path simulation-model:model.validation.status ],
+        [ skos:example "PMSF FMIBench 1.9.9.4 using osi3test 0.6.0 on Windows 10 1809" ;
+            sh:datatype xsd:string ;
+            sh:description "Specification of simulation environment used for model validation."@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of model.validation.platform failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "model.validation.platform"@en ;
+            sh:order 5 ;
+            sh:path simulation-model:model.validation.platform ],
+        [ skos:example "did:web:registry.gaia-x.eu:Validation:T9rkdS19poUrbbAEk3rXMqZ3hzVezwDrrxaf" ;
+            sh:datatype xsd:string ;
+            sh:description "Link to model validation report, if any exists"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of model.validation.report failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "model.validation.report"@en ;
+            sh:order 6 ;
+            sh:path simulation-model:model.validation.report ],
+        [ skos:example "http://mywebsite.de/ValidationConcept.md" ;
+            sh:datatype xsd:string ;
+            sh:description "Link to document detailing the validation concepts used for model validation, if any exists"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of model.validation.concept failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "model.validation.concept"@en ;
+            sh:order 7 ;
+            sh:path simulation-model:model.validation.concept ],
+        [ skos:example "Potentially real-time capable, if less than 100 objects in object list input at any one time" ;
+            sh:datatype xsd:string ;
+            sh:description "Description of model performance characteristics."@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of model.implementation.performance-characteristics failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "model.implementation.performance-characteristics"@en ;
+            sh:order 8 ;
+            sh:path simulation-model:model.implementation.performance-characteristics ] ;
+    sh:targetClass simulation-model:Quality .
+
+simulation-model:DataSourceShape a sh:NodeShape ;
```

Comment:
Yes I agree with you. I just added the attribute since it was the closest from my understanding for something like the _origin_ of a model. I added it under `DataSourceShape` since I don't have any clue for other attributes in that part. However, it would be fine to just keep it empty for me. 

@jtdemer should we reference a specification document now from the `market-place` ontology? I could integrate a change for the bundled data also in the OSITrace in the other currently open PR #129

Reply:
I think the specification should probably be referenced from here because IMO it belongs strongly to the model and is not just an optional thing in the bundle. But maybe it is possible to reuse some structure to reference it (e.g. LinkShape)?

Reply:
A reuse sounds nice. However, the `LinkShape` doesn't exist anymore, right?

Reply:
The dataSource-shape is defined in the domains hdmap and ositrace and should stay there for the described purpose.

About the other points:
In my opinion, we should only use bundleData for linking to other registered assets. I.e. there is another domain GPS tracks defined with metadata. This asset is registered in the ecosystem and can then be linked to a new asset via Bundle data using the DID of the registered asset. 

Bundle data is not suitable for adding a file to an asset. If you cannot transfer it to a new domain, then you should define it in the asset domain in which additional files are required. For an overview you can look in the [sample asset structure](https://github.com/GAIA-X4PLC-AAD/hd-map-asset-example/tree/main/asset). We defined there the categories data, metadata, visualization and validation. One should suite also for you.

So two ways are possible:
a) new asset domain for addional data
b) included in the current asset describtion
-> The asset structure ist than described in the corresboning manifest file: https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/manifest

Reply:
@heuerfin I think this is in this marketplace shape now. 

This was not about removing DataSourceShape from OSITrace or hdmap. My question was if its content here (specification document) fits in the general usage of the DataSourceShape or if this shouldn't be called differently.

Also I wanted to start the discussion if something more generic and reusable exists to reference a document that's part of the actual asset like a specification or log file etc. I guess we have this problem more often also for other shapes because so far we used bundle data for this as we apparently did not get how bundle data is supposed to be used.

Reply:
Maybe to clear some things up :D this pull request is in regards to simulation-models (e.g. the radar model). What I got from Thomas question is: I placed an attribute called `model.specification` in the domain of `DataSource`. However, a simulation model doesn't have any `Data` where it originates from. So there are in my opinion two options:

1. Move `model.specification` to another shape, e.g. `ContentShape` (leaves `DataSourceShape' empty for a simulation model)
2. Keep `model.specification` in `DataSource` (but a model specification is not really a data source)

Reply:
We already have added an example [here](https://github.com/GAIA-X4PLC-AAD/sensordata/tree/main/ars548/sampledata/119_pmsf_adc_cutout). This could be extended by a manifest file and the bundle data issue should probably be fixed. But we should probably discuss this in the other repo.

I also did not propose to have DataSourceShape in a more generic way. I did not talk about changing anything concerning DataSourceShape in OSITrace or hdmap. My only point here was that the use of the name "DataSource" **in this PR** for the contained content might not fit in terms of harmonization with the content of DataSourceShape in OSITrace and hdmap, not in terms of attribute harmonization but actual meaning harmonization: Should a specification document be considered a "data source" for a simulation model?

But as I not understood, the specification document should not be referenced here at all but rather in the manifest? Is this correct? So this should be completely removed?

Reply:
Honestly I'm not sure. I think we will clear things up in the next meeting